internal/api/front/perses/package.json

```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve --hmr",
-    "build": "ng build --prod",
+    "build": "ng build --prod --base-href=/ui/",
```

Comment:
Because I prefixed the UI by /ui into the Golang code. Like that I find it clearer : /ui go to the ui /api go to the api.  

We can redirect the / to /ui as well

Reply:
yeah I think it can be removed. api is prefixed by /api, the rest goes to the UI

Reply:
removed